src/switch_core_media.c

```diff
@@ -6550,6 +6554,12 @@ SWITCH_DECLARE(uint8_t) switch_core_media_negotiate_sdp(switch_core_session_t *s
 								switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session), SWITCH_LOG_DEBUG,
```

Comment:
I think looking at the DIFF in the ticket, this line should be moved below the if statement. Might be confusing to log that it is a near match then afterwards it was rejected as not a near match

Reply:
You are right, thanks for pointing it out.